core/app.ts

```diff
@@ -12,17 +12,20 @@ import EventBus from '@vue-storefront/core/plugins/event-bus'
 import { registerTheme, plugins, mixins, filters } from '@vue-storefront/core/lib/themes'
 import registerExtensions from '@vue-storefront/core/lib/extensions'
 import i18n from '@vue-storefront/core/lib/i18n'
-import shippingMethods from '@vue-storefront/core/resource/shipping_methods.json'
 
 import store from '@vue-storefront/store'
 import coreModules from '@vue-storefront/store/modules'
 import { prepareStoreView } from '@vue-storefront/store/lib/multistore'
 
-import App from 'theme/App'
+import App from 'theme/App.vue'
```

Comment:
shouldn't it be resolved by default without `vue` (look at extensions in webpack config)